bin/shpec

```diff
@@ -74,11 +74,11 @@ assert() {
     print_result "[ $2 -lt $3 ]" \
       "Expected [$2] to be < [$3]"
   ;;
-  ( xmatch )
+  ( xglob )
     print_result "case '$2' in *$3*) :;; *) false;; esac" \
```

Comment:
@schorsch3000 I would prefer if we did away with the default globbing here; there should be no (potentially unexpected) behavior beyond what is passed to the matcher.

That is, `assert glob "partially" "partial"` should fail; if users want that behavior, they should use `assert grep "partially" "partial"` instead.